contracts/Rollup.sol

```diff
@@ -1,16 +1,16 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.6.1;
 
-import '../node_modules/openzeppelin-solidity/contracts/ownership/Ownable.sol';
+import '../node_modules/@openzeppelin/contracts/ownership/Ownable.sol';
 import './lib/RollupHelpers.sol';
 import './RollupInterface.sol';
 import './VerifierInterface.sol';
 
 /**
  * @dev Define interface ERC20 contract
  */
-contract ERC20 {
-    function transfer(address recipient, uint256 amount) external returns (bool);
-    function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
+abstract contract ERC20 {
+    function transfer(address recipient, uint256 amount) external virtual returns (bool);
```

Comment:
as it says in docs: "Add virtual to every non-interface function you intend to override" so if we defined as an interface we shouldn't use the `virtual` key word

Reply:
change `abstract contract` to `interface`